bigtable-hbase/src/test/resources/com/google/cloud/bigtable/hbase/bigtable-hbase.properties

```diff
@@ -0,0 +1 @@
+bigtable.hbase.version=${project.artifactId}-${project.version}
```

Comment:
It is not used yet, please don't add it. This is file is not filtered as the one in main/resources